python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -56,7 +56,8 @@ async def complete_chat(
         settings.messages = self._prepare_chat_history_for_request(chat_history)
         settings.stream = False
```

Comment:
I see the issue now, in almost all integration this piece is here after the messages:
`
        if not settings.ai_model_id:
            settings.ai_model_id = self.ai_model_id
`
Please add that and then remove the overridden prepare_settings_dict form the prompt settings file.